dlt/helpers/streamlit_app/dashboard.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+
+import dlt
+import streamlit as st
+from dlt.cli import echo as fmt
+from dlt.helpers.streamlit_app.blocks.query import maybe_run_query
+from dlt.helpers.streamlit_app.blocks.table_hints import list_table_hints
+from dlt.helpers.streamlit_app.menu import menu
+from dlt.helpers.streamlit_app.widgets import schema_picker
+from dlt.pipeline import Pipeline
+
+
+def write_data_explorer_page(
```

Comment:
why is this not a page?

Reply:
the reason for this is that streamlit doesn't have an option to select a default page in multipage apps thus I had to put it in the main entrypoint file 😞